src/ainft.ts

```diff
@@ -151,22 +152,34 @@ export default class AinftJs {
   }
 
   /**
-   * Opens a session with the blockchain endpoint.
+   * Connects to the blockchain endpoint.
    */
-  async open() {
-    const privateKey = this.ain.wallet.defaultAccount?.private_key;
-    if (privateKey) {
-      await this.ainize.login(privateKey);
-    } else {
-      await this.ainize.loginWithSigner();
+  async connect() {
+    if (!this.isConnected()) {
```

Comment:
how about throwing some exception for the other case?

Reply:
okay, done!